### ISSUE TITLE
"Same as Billing Address" Checkbox not always visible in Admin Create Order Screen

### ISSUE THREAD

**Issue by gwharton** [magento/magento2#30993]
### Preconditions (*)

magento 2.4.1

### Steps to reproduce (*)

```
- BACKEND

-- Customers
--- All Customers
---- Add New Customer
----- First Name = Test
----- Last Name = Test
----- Email = test@test.com
----- Save Customer

-- Sales Orders
--- Create New Order
---- Select "Test Test" customer
```

Look at the Shipping Address Section.
Verify that the checkbox to select "Same As Billing Address" is visible.
Refresh the screen
Verify that the checkbox to select "Same As Billing Address" is visible after screen refresh

### Expected result (*)

![image](https://user-images.githubusercontent.com/30697781/99750114-b3559e00-2ad7-11eb-920f-de4a7462094f.png)

### Actual result (*)

Checkbox is not visible
![image](https://user-images.githubusercontent.com/30697781/99750149-c4061400-2ad7-11eb-9491-32a629cfba5f.png)

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by krishprakash** [comment]
@magento give me 2.4-develop instance

**Comment by gwharton** [comment]
A similar issue seems to apply to the radio boxes for payments and shipping. Sometimes they are not visible, but clicking on the text label causes them to appear.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @krishprakash 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-39301` were created

**Issue Available**: @krishprakash, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by gwharton** [comment]
not stale

**Comment by srsathish92** [comment]
@gwharton Not able to reproduce in 2.4-develop branch, Please verify and close the ticket.

**Comment by gwharton** [comment]
Agree, not reproducable in 2.4.2 onwards. I can still reproduce the same issue with the payment and shipping checkboxes so I will create a separate issue for those.

**Comment by magento-engcom-team** [comment]
Hi @gwharton. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-39301` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.

**Comment by simonmaass** [comment]
we have this issue in Magento ver. 2.4.6-p3
